docs/quick-start.md

```diff
@@ -33,10 +33,13 @@ All Netlify CMS files are contained in a static `admin` folder, stored at the ro
 These generators ... | store static files in
 --- | ---
 Jekyll, GitBook | `/` (project root)
-Hugo | `/static`
+Hugo, Gatsby* | `/static`
 Hexo, Middleman | `/source`
 Spike | `/views`
 
+Notes:
+- Gatsby treats `static` folder more strictly and will not render the admin page as the others. You will have to make a [page component](https://www.gatsbyjs.org/docs/building-with-components/) contining the necessary contents of the admin page. However, the `config.yml` is to be placed in the `static` folder in the same way. 
+
```

Comment:
A couple quick typo fixes:
* "Gatsby treats _the_ `static` folder..." (add "the")
* "_containing_ the necessary contents..." (fix spelling of "containing")

Otherwise LGTM!